routes/addCustomer.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Created by johnschroeder on 7/7/15.
+ */
+
+var express = require("express");
+var Q = require('q');
+var router = express.Router();
+
+
+/*
+ Usage:
+ localhost:50001/AddCustomer/customerName/
+ */
+
+router.route("/:customerName").get(function(req, res) {
+    var db = require("../imp_services/impdb.js").connect();
+
+    //Q.longStackSupport = true;   // for error checking
+
+    Q.fcall(db.beginTransaction())
+        .then(db.query("USE " + db.databaseName))
+        .then(db.query("CALL AddCustomer ('" + req.params.customerName + "')"))
+        // TODO: log this entry?
+        .then(function(rows, columns){
+            console.log("Success");
```

Comment:
One of the nice things is how robust our mysql logging is now. I think we can probably leave this in for now since we already need a ticket soon for adding logging to every route now that logging will be in.